wled00/FX.cpp

```diff
@@ -6466,6 +6466,113 @@ uint16_t mode_matripix(void) {                  // Matripix. By Andrew Tuline.
 static const char _data_FX_MODE_MATRIPIX[] PROGMEM = "Matripix@!,Brightness;!,!;!;1v;ix=64,m12=2,si=1"; //,rev=1,mi=1,rY=1,mY=1 Circle, WeWillRockYou, reverseX
 
 
+//////////////////////
+// *    WIPE OUT    //
+//////////////////////
+uint16_t mode_wipe_out(void) {
+  if (SEGLEN == 1) return mode_static();
+
+  uint16_t duration = 1000 - (SEGMENT.speed * 4); // Adjust speed (1-255) to duration (1000ms to 20ms)
+  uint16_t stepDuration = duration / SEGLEN;
+  uint32_t currentTime = millis();
+
+  if (SEGENV.call == 0) {
+    SEGENV.aux0 = currentTime; // Start time
+    SEGENV.aux1 = 0; // Current LED
+  }
+
+  if (currentTime - SEGENV.aux0 > stepDuration) {
+    SEGENV.aux0 = currentTime;
+    if (SEGENV.aux1 < SEGLEN) {
+      SEGMENT.setPixelColor(SEGENV.aux1, BLACK);
+      SEGENV.aux1++;
+    } else {
+      // All LEDs are off, reset
+      SEGENV.aux1 = 0;
+      strip.setBrightness(0);
+      return 65535;
```

Comment:
this will not cause a "forever black" - the effect will run again in 65535 millis = 65 seconds.  Also manipulation `strip` properties in an effect (at segment level) is a very fragile thing and should be avoided.